dscache/dscache_test.go

```diff
@@ -69,3 +74,81 @@ func TestDscacheAssignSaveAndLoad(t *testing.T) {
 		t.Errorf("expected, 2 refs, got %d refs", loadable.Root.RefsLength())
 	}
 }
+
+func TestResolveRef(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer os.RemoveAll(tmpdir)
+
+	ctx := context.Background()
+	fs := localfs.NewFS()
+	path := filepath.Join(tmpdir, "dscache.qfb")
+	dsc := NewDscache(ctx, fs, nil, path)
+
+	dsrefspec.ResolverSpec(t, dsc, func(r dsref.Ref, author identity.Author, log *oplog.Log) error {
```

Comment:
For this situation where the callback doesn't use all of its parameters, could you rename them to `_`?